expenses/reports.py

```diff
@@ -80,6 +80,39 @@ class TextFieldOption(Option):
     type: str = attr.ib("text", init=False, repr=False)
 
 
+class ReportItemFormatter:
+    def format_money(self, amount: typing.Union[int, float, decimal.Decimal]) -> str:
+        return format_money(amount)
+
+    def format_category(self, c: Category) -> str:
+        return c
+
+    def format_link(self, vendor: str) -> str:
+        return vendor
+
+
+class CsvFormatter(ReportItemFormatter):
+
+    """"Remove breaking space from formatted string"""
```

Comment:
1. A docstring should describe the entire class, not the specific task of a method
2. I would prefer to put just the number directly in the CSV file, maybe formatted according to region / with 2 decimal places (so 1,00 would be fine for Polish, no need for the currency symbol)